bin/yaml/libraries.yaml

```diff
@@ -377,9 +377,17 @@ libraries:
       - 2.3.8
       type: github
     eastl:
+      after_stage_script:
+      - mkdir build
+      - cd build
+      - /opt/compiler-explorer/cmake/bin/cmake ..
```

Comment:
It is slightly unfortunate that we now run `cmake` even on the versions for which it is not necessary (the ones not using `FetchContent`), hopefully this isn't a big deal but I'm happy to hear about alternatives if anyone thinks of one.

Reply:
Which versions? We can solve that by making a new section and only applying the `after_stage_script` to those. I'll try out in a branch and merge if I can get it to work.

Reply:
Hi @mattgodbolt,
> We can solve that by making a new section and only applying the after_stage_script to those.

Thanks for your comment, I've changed the code a bit follow what `benchmark` is doing and this seems to work properly without doing any unnecessary work. I'm not sure if that's exactly what you were suggesting, or if there's a better way, let me know what you think.